Asterix/wfsc/wf_control_functions.py

```diff
@@ -628,9 +785,12 @@ def calc_steepest_solution(mask, Result_Estimate, Hessian_Matrix, Jacobian, test
          voltage to apply on each deformable mirror actuator
    
     """
+    # probably not working in polychromatic yet
+    if len(Result_Estimate) > 1:
+        raise Exception("steepest correction does not workin polychromatic")
```

Comment:
`ValueError`

Reply:
And you're missing a space in the string. Would also be nicer if you make this a real sentence: capitalize the first word and put a period at the end.